src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -417,6 +424,135 @@ bool emberAfOperationalCredentialsClusterUpdateNOCCallback(EndpointId endpoint,
     return true;
 }
 
+bool emberAfOperationalCredentialsClusterCertChainRequestCallback(EndpointId endpoint, app::CommandHandler * commandObj,
+                                                                  uint16_t certChainType)
```

Comment:
`certChainType` should be renamed to `certificateType` --> need rename in XML. Please review the Opcreds cluster XML for spec-compliant names.

Reply:
will update